groovy-language-server/src/main/java/com/palantir/ls/groovy/GroovyTextDocumentService.java

```diff
@@ -208,13 +210,8 @@ private void publishDiagnostics(Set<Diagnostic> diagnostics) {
         config.getPublishDiagnostics().accept(paramsBuilder.build());
     }
 
-    private Path resolveUriToWorkspaceRoot(String uri) {
-        Path filePath = Paths.get(uri);
-        return filePath.isAbsolute() ? filePath : provider.get().getWorkspaceRoot().resolve(uri).toAbsolutePath();
-    }
-
-    private void assertFileExists(Path uri) {
-        checkArgument(uri.toFile().exists(), String.format("Uri '%s' does not exist", uri));
+    private void assertFileExists(URI uri) {
+        checkArgument(Paths.get(uri).toFile().exists(), String.format("Uri '%s' does not exist", uri));
```

Comment:
checkArgument(new File(uri).exists(), String.format("Uri '%s' does not exist", uri));
https://docs.oracle.com/javase/7/docs/api/java/io/File.html#File(java.net.URI)